Assets/MixedRealityToolkit.Services/InputSystem/Editor/NearInteractionTouchableInspector.cs

```diff
@@ -118,4 +90,36 @@ public override void OnInspectorGUI()
             }
         }
     }
-}
\ No newline at end of file
+
+    public class NearInteractionTouchableInspectorBase : UnityEditor.Editor
```

Comment:
can you add a  [UnityEditor.CustomEditor(typeof(BaseNearInteractionTouchable), true)] here so this is the fallback for all the NearInteractionTouchables? Then you also won't have to define the NearInteractionTouchableUnityUI editor further up.